src/TwoSlotDisplay.js

```diff
@@ -34,11 +42,13 @@ class TwoSlotDisplay extends Component {
 
   render() {
     let next = getNextSlot(this.state.day, this.state.slot);
+    const weekend = this.state.day === 0 || this.state.day === 6;
```

Comment:
I guess here we have an issue. As per line no. 20, you decrease the day count by 1 if it's not a weekend. So, for say it is 1 and after decrease it becomes 0. Then when in render you check, it'll be shown as weekend. If I'm not wrong, this is not the behaviour that we wish to achieve.

Please make me aware if I'm wrong here.

Reply:
Actually the `getDay()` function returns integers ranging from **0** to **6** for **Sunday** to **Saturday**. So in either case, the state is set to **0**. 
However, in `Constants.js`, the days are mapped as `[ 0, 4 ] => [ 'Monday', 'Friday' ]`. 
Hence, for weekdays,  we need to reduce the output from the `getDay()` function by 1.

But I see what will happen for Mondays. I'll fix it.
The `this.state.day === 6` is useless. I will fix that in next commit.